app/swagger/swagger/requests/letters_generator.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+# app/controllers/v0/letters_generator_controller.rb
+module Swagger
+  module Requests
+    class LettersGenerator < ApplicationController
+      include Swagger::Blocks
+
+      swagger_path '/v0/letters_generator' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Get a list of available letter types that a user can generate and download'
+          key :tags, %w[letters_generator]
+          key :operationId, 'getLetters'
+          key :summary, 'Get a list of available letter types that a user can generate and download'
+
+          response 200 do
+            key :description, 'Letters Generator metadata list'
+
+            schema do
+              key :type, :object
+
+              property :letters do
+                key :type, :array
+                items do
+                  key :type, :object
+                  key :required, %i[name letter_type]
+                  property :name, type: :string, example: 'Proof of Service Letter'
+                  property :letter_type, type: :string, enum: %w[
+                    commissary
+                    proof_of_service
+                    medicare_partd
+                    minimum_essential_coverage
+                    service_verification
+                    civil_service
+                    benefit_summary
+                    benefit_summary_dependent
+                    benefit_verification
+                    certificate_of_eligibility
+                  ], example: 'proof_of_service'
+                end
+              end
+
+              property :letters_destination do
+                key :type, :array
+                items do
+                  key :type, :object
+                  property :country, type: :string, example: 'United States'
+                  property :addressLine1, type: :string, example: '555 Generic Blvd'
+                  property :addressLine2, type: :string
+                  property :addressLine3, type: :string
+                  property :city, type: :string, example: 'Springfield'
+                  property :state, type: :string, example: 'VA'
+                  property :zipCode, type: :string, example: '90210'
+                  property :fullName, type: :string, example: 'Jane Doe'
+                end
+              end
+
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/letters_generator/beneficiary' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Get a benificiary\'s military service and benefits information'
+          key :tags, %w[letters_generator]
+          key :operationId, 'getBeneficiaryInfo'
+          key :summary, 'Get a benificiary\'s military service and benefits information'
+
+          response 200 do
+            key :description, 'Letters Generator beneficiary response'
+
+            schema do
```

Comment:
Soooo instead of typing these all out by hand, you can refer to an external schema like this:

```yml
schema:
  $ref: "https://api.va.gov/internal/docs/va-letter-generator/v1/openapi.json#/components/schemas/LettersResponse"
```